ccm_web/client/src/pages/BulkSectionCreate.tsx

```diff
@@ -0,0 +1,505 @@
+                <Typography>Correct the file and{renderUploadAgainButton()}</Typography>
+              </Paper>
+            </Grid>
+          </Box>
+        </Grid>
+      </div>)
+  }
+
+  const renderTopLevelErrors = (errors: JSX.Element[]): JSX.Element => {
+    return (
+      <div>
+        {renderCSVFileName()}
+        <Grid container justify='flex-start'>
+          <Grid item xs={12} className={topLevelClasses.dialog}>
+            <Paper role='alert'>
+              <Typography>Review your CSV file</Typography>
+              <ErrorIcon className={topLevelClasses.dialogIcon} fontSize='large'/>
+              <Typography>Correct the file and{renderUploadAgainButton()}</Typography>
+              <ol>
+                {errors.map(e => {
+                  return (<li key={e.key}>{e}</li>)
+                })}
+              </ol>
+            </Paper>
+          </Grid>
+        </Grid>
+      </div>)
+  }
+
+  const renderInvalidUpload = (): JSX.Element => {
+    let rowLevelErrors: JSX.Element | undefined
+    let schemaLevelErrors: JSX.Element | undefined
+    if (pageState.rowInvalidations.length > 0) {
+      rowLevelErrors = renderRowLevelErrors(pageState.rowInvalidations)
+    }
+    if (pageState.schemaInvalidation.length > 0) {
+      const schemaErrors: JSX.Element[] = pageState.schemaInvalidation.map((invalidation, i) => {
+        return (<div key={i}>{invalidation.error}</div>)
+      })
+      schemaLevelErrors = <div>{renderTopLevelErrors(schemaErrors)}</div>
+    }
+    return (
+      <div>
+        {schemaLevelErrors !== undefined && schemaLevelErrors}
+        {rowLevelErrors !== undefined && rowLevelErrors}
+      </div>
+    )
+  }
+
+  const renderAPIError = (): JSX.Element => {
+    return (
+      <Grid item xs={12} className={apiErrorClasses.dialog}>
+        <Paper role='alert' >
+          <ErrorIcon className={apiErrorClasses.dialogIcon} fontSize='large'/>
+          <Typography>Sometheing went wrong.  Please try again later.</Typography>
+          {renderTryAgainButton()}
+        </Paper>
+      </Grid>
+    )
+  }
+
+  const renderConfirm = (sectionNames: Section[]): JSX.Element => {
+    return (
+      <div>
+        {renderCSVFileName()}
+        <Grid container>
+          <Box clone order={{ xs: 2, sm: 1 }}>
+            <Grid item xs={12} sm={9} className={confirmationClasses.table}>
+              <BulkSectionCreateUploadConfirmationTable sectionNames={sectionNames} />
+            </Grid>
+          </Box>
+          <Box clone order={{ xs: 1, sm: 2 }}>
+            <Grid item xs={12} sm={3} className={confirmationClasses.dialog}>
+              <Paper role='status'>
+                <Typography>Review your CSV file</Typography>
+                <CloudDoneIcon className={confirmationClasses.dialogIcon} fontSize='large'/>
+                <Typography>Your file is valid!  If this looks correct proceed with download</Typography>
```

Comment:
@chrisrrowland after the file upload pass the validation, we don't download anything, we make a POST request to BE with the parsed CSV data and receives as response of section creation. This message could be `Your file is valid!  If this looks correct will proceed sections creation`

Reply:
Right, I didn't do anything with the backend part for submitting as noted in the PR, but I can update the text.  

Also note "Do something" is not the final text for the button 🤣

Reply:
I know.. you mentioned it is a place holder button. I am trying to make the code as relavent as possible with BE not being ready